databases/week3/Week3Homework.sql

```diff
@@ -0,0 +1,205 @@
+SET NAMES utf8mb4;
+
+CREATE DATABASE meals_sharing;
+use meals_sharing;
+
+
+-- create table meals
+
+CREATE TABLE `meal`(
+ `id` int(10) unsigned NOT NULL auto_increment PRIMARY KEY,
+ `title` varchar(255) NOT NULL,
+ `description`text NULL default NULL,
+ `location` varchar(255) NOT NULL,
+ `When` datetime NOT NULL,
+ `max_reservations` int(10) unsigned NOT NULL,
+ `price` decimal(15,2),
+ `created_date` DATE NOT NULL
+ );
+ 
+ DROP TABLE meal;
+ 
+ CREATE TABLE `meal`(
+ `id` int(10) unsigned NOT NULL auto_increment PRIMARY KEY,
+ `title` varchar(255) NOT NULL,
+ `description`text NULL default NULL,
+ `location` varchar(255) NOT NULL,
+ `When` datetime NOT NULL,
+ `max_reservations` int(10) unsigned NOT NULL,
+ `price` decimal(15,2),
+ `created_date` DATE NOT NULL
+ )ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+ 
+ -- create table reservation
+ 
+ CREATE TABLE `reservation`(
+ `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+ `number_of_guests` int(10) unsigned NOT NULL,
+ `meal_id` int(10) unsigned NOT NULL,
+ `created_date` DATE NOT NULL,
+ `contact_phonenumber` varchar(255) NOT NULL,
+ `contact_name` varchar(255) NOT NULL,
+  CONSTRAINT `fk_meal_reservation` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+  )ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```

Comment:
Here in table reservation you forgot one column, please add it and update all queries related to it.

Reply:
Yes, added now